src/Parsers/ASTIdentifier.cpp

```diff
@@ -111,6 +111,16 @@ void ASTIdentifier::resetTable(const String & database_name, const String & tabl
     uuid = ident.uuid;
 }
 
+void ASTIdentifier::updateTreeHashImpl(SipHash & hash_state) const
+{
+    hash_state.update(name.data(), name.size());
```

Comment:
It's incorrect.
Suppose we have

name: a
alias: bc

or

name: ab
alias: c

it will be hashed identical.
But should not.